inscriptions/ISic003523.xml

```diff
@@ -192,38 +205,24 @@
     </facsimile>
     <text>
         <body>
-            <div type="edition" xml:space="preserve" subtype="EDR" n="EDR033617" xml:lang="la">
+            <div type="edition" xml:space="preserve" xml:lang="la" resp="#DEE">
                 <ab>
-                    <lb n="1"/>
-                    Dis Manibus<lb n="2"/>Cytisi.<lb n="3"/>Vixit annis X;<lb n="4"/>fecit Stellius
-                    <expan>
-                        <abbr>Novell</abbr>
-                        <ex>us</ex>
-                    </expan>
-                    <lb n="5"/>Amaranthus filio<lb n="6"/>benemerenti.
-                </ab>
-                <!--This EDR text 033617 originally came via the Europeana EAGLE project, http://www.eagle-network.eu/-->
-            </div>
-            <div type="edition" xml:space="preserve" subtype="PHI" n="PHI313645">
-                <ab>
-                    <lb n="1"/>
-                    <surplus>²ramus</surplus>² Dis Manibus <surplus>²ramus</surplus>²<lb n="2"/> Cytisi.<lb n="3"/>
-                    vixit annis X.<lb n="4"/>fecit Stellius
-                    <expan>
-                        <abbr>Novell</abbr>
-                        <ex>us</ex>
-                    </expan>
-                    <lb n="5"/>Amaranthus filio<lb n="6"/> bene merenti.
-                    <lb n="7"/>
-                    <surplus>²ramus</surplus>² <surplus>²ramus</surplus>²
-                    <lb n="8"/>
+                    <lb n="1"/><g ref="#ramus"/> Dis Manibus <g ref="#ramus"/>
+                    <lb n="2"/>Cytisi
+                    <lb n="3"/>vixit annis <num value="10">X</num>
```

Comment:
In the photograph, I can see a line above the X, so `<hi rend="supraline">X</hi>` inside the `<num>`